### DISCUSSION TITLE
is Quota Monitor For AWS a Viable Solution for EC2 Quota Monitoring/Alerts

### DISCUSSION THREAD

**Discussion by naut-brian** [aws-solutions/quota-monitor-for-aws#174] [Q&A] [answered]
Hi,
I'm looking for a solution that will be able to send notifications when a single instance type quota (like G series quota, P Series quota, etc) is about to be full. Is this project a viable solution for this? Some other solutions simply pool together all the EC2 quotas which is not very useful.

**Comment by abewub** [comment, marked as answer]
The solution monitors all available quotas, but you can narrow down the notifications to in this [configuration](https://docs.aws.amazon.com/solutions/latest/quota-monitor-for-aws/step-6.-configure-notifications-optional.html). But it's an exclusion list rather than an allow list, so it doesn't fit neatly what you are looking for. You can specify the [notification threshold](https://docs.aws.amazon.com/solutions/latest/quota-monitor-for-aws/architecture-overview.html), which by default is 80% utilization.

**Reply by naut-brian** [reply]
Oh I think this will do what I need it to do then. I'm okay with all quotas being included as long as it warns when a single ec2 instance quota is about to reach the threshold
